### PR TITLE
feat(atom/badge): add design feature on AtomBadge component

### LINKED ISSUE #1298: Component Badge - Iteration request: Add a new Prop "Design"
**Is your iteration request related to a problem?**
We need more flexibility for the component Badge. 
Specifically a design with solid "soft" is needed.

**Describe the solution you'd like**
Specific colour combination for the new Design "soft" will be provided below:

We suggest a new Prop "design", with a default value with what we currently have, and a new one: "Soft"

**Additional context**

- Component: https://sui-components.now.sh/workbench/atom/badge/demo
- Code of the component: https://github.com/SUI-Components/sui-components/tree/master/components/atom/badge

### PR DESCRIPTION
**Pull request by AitorRodriguez990**
ISSUES CLOSED: #1298

## ATOM/BADGE
**TASK**: Private JIRA.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description, Motivation and Context

We need AtomBadge to have some soft versions.

### Screenshots - Animations

image

### CODE REVIEW

**Review by emiliovillu — APPROVED** [review]

**Comment by andresin87** [comment]
doesn't matter if its repeated. Cuold you please add the design solid in the design demo? Good job.

Neutral type is ok @turolopezsanabria? It looks too much dark 4 me
[https://sui-components-q7ul2mc3o.vercel.app/](https://sui-components-q7ul2mc3o.vercel.app/)

**Review by javiauso — APPROVED** [review]

File: components/atom/badge/src/index.js [hunk 1]
```diff
@@ -116,4 +127,9 @@ AtomBadge.defaultProps = {
 }
 
 export default AtomBadge
-export {TYPES as atomBadgeTypes, SIZES as atomBadgeSizes}
+export {}
```
Comment (klaufel) [inline components/atom/badge/src/index.js]: 👀

**Review by oriolpuig — APPROVED** [review]

**Review by klaufel — APPROVED** [review]

**Review by danivm — APPROVED** [review]

**Review by turolopezsanabria — APPROVED** [review]
👏 👏 está genial muchas gracias 
I'm checking the colour for Neutral, but this is NOT a blocker if you need to merge now.
Thnx

**Review by jordevo — APPROVED** [review]

File: components/atom/badge/src/index.scss [hunk 2]
```diff
@@ -53,9 +53,40 @@ $badge-transparent: (
...
+  info: (
+    bgc: $c-gray-light-4,
+    c: $c-gray
+  ),
+  new: (
+    bgc: $c-accent-light-4,
+    c: $c-accent
+  ),
+  neutral: (
+    bgc: $c-gray-dark-3,
```
Comment (turolopezsanabria) [inline components/atom/badge/src/index.scss]: please change neutral values to
bgc: $c-gray-light-4,
c: $c-gray

will be the same as info, but it's ok
thanks

Comment (andresin87) [reply components/atom/badge/src/index.scss]: When the color is reviewed, it can be integrated

File: components/atom/badge/src/index.js (hunk 1 above)
Comment (AitorRodriguez990) [reply components/atom/badge/src/index.js]: Fixed 😉

File: components/atom/badge/src/index.scss (hunk 2 above)
Comment (AitorRodriguez990) [reply components/atom/badge/src/index.scss]: Done!

**Review by andresin87 — APPROVED** [review]

**Pull request merged**